erigon-lib/kv/kvcache/cache_test.go

```diff
@@ -171,7 +171,7 @@ func TestEviction(t *testing.T) {
 }
 
 func TestAPI(t *testing.T) {
-	if runtime.GOOS == "windows" {
+	if runtime.GOOS != "linux" {
 		t.Skip("fix me on win please")
```

Comment:
this text should also be changed

Reply:
it's part of another PR (which makes CI green)